packages/@tinacms/react-toolbar/src/components/Toolbar.tsx

```diff
@@ -62,6 +64,16 @@ export const Toolbar = () => {
   // this is used to refreshe the discard button to fix it not updating when pressed after the page loads
   const [, setState] = React.useState(0)
 
+  // Global plugin hamburger menu
+  const screens = cms.plugins.getType<ScreenPlugin>('screen')
+  useSubscribable(screens)
+  const allScreens = screens.all()
+  const showMenu = allScreens.length > 0
+  const [activeScreen, setActiveView] = React.useState<ScreenPlugin | null>(
+    null
+  )
+  const [menuIsVisible, setMenuVisibility] = React.useState(false)
```

Comment:
Maybe rename to `menuIsOpened`?

`showMenu` and `setMenuVisibility` sound really similar